sanic/app.py

```diff
@@ -117,6 +117,21 @@ def decorator(listener):
 
         return decorator
 
+    def register_listener(self, listener, event):
+        """
+        Register the listener for a given event.
+
```

Comment:
can you use single spacing here in the doc string?

Reply:
Removed erroneous whitespace in a new commit